src/test/java/mil/dds/anet/test/resources/MergePeopleTest.java

```diff
@@ -0,0 +1,708 @@
+        variables);
+    assertThat(nrUpdated).isEqualTo(1);
+    return getPerson(person.getUuid());
+  }
+
+  private Person getPerson(String personUuid) {
+    return graphQLHelper.getObjectById(admin, "person", PERSON_FIELDS, personUuid,
+        new TypeReference<GraphQlResponse<Person>>() {});
+  }
+
+  private void removePositionPerson(Position position) {
+    Integer nrDeleted =
+        graphQLHelper.deleteObject(admin, "deletePersonFromPosition", position.getUuid());
+    assertThat(nrDeleted).isEqualTo(1);
+  }
+
+  private Person mergePeople(String loserUuid, String winnerUuid, Boolean copyPosition) {
+    Map<String, Object> variables = new HashMap<>();
+    variables.put("winnerUuid", winnerUuid);
+    variables.put("loserUuid", loserUuid);
+    variables.put("copyPosition", copyPosition);
+    Integer nrUpdated = graphQLHelper.updateObject(admin,
+        "mutation ($winnerUuid: String!, $loserUuid: String!, $copyPosition: Boolean!) { payload: mergePeople (winnerUuid: $winnerUuid, loserUuid: $loserUuid, copyPosition: $copyPosition) }",
+        variables);
+    assertThat(nrUpdated).isEqualTo(1);
+    return graphQLHelper.getObjectById(admin, "person", PERSON_FIELDS, winnerUuid,
+        new TypeReference<GraphQlResponse<Person>>() {});
+  }
+
+  @Test
+  public void loserHasPositionWinnerHasNoPositionCopyPositionNotSelected() {
+    Position positionLoser = createPosition("Loser Position One");
+    // Active peoplePosition without person must be created when the position is created.
+    assertThat(positionLoser.getPreviousPeople().stream()
+        .filter(f -> Objects.isNull(f.getEndTime())).count()).isEqualTo(1);
+    assertThat(positionLoser.getPreviousPeople().stream()
+        .filter(f -> Objects.isNull(f.getEndTime()) && Objects.isNull(f.getPersonUuid())).count())
+            .isEqualTo(1);
+
+    Person personLoser = createPerson("Loser Has Position");
+    // Active peoplePosition without position must be created when the person is created.
+    assertThat(personLoser.getPreviousPositions().stream()
+        .filter(f -> Objects.isNull(f.getEndTime())).count()).isEqualTo(1);
+    assertThat(personLoser.getPreviousPositions().stream()
+        .filter(f -> Objects.isNull(f.getEndTime()) && Objects.isNull(f.getPositionUuid())).count())
+            .isEqualTo(1);
+
+    Person personWinner = createPerson("Winner Has No Position");
+    // Active peoplePosition without position must be created when the person is created.
+    assertThat(personWinner.getPreviousPositions().stream()
+        .filter(f -> Objects.isNull(f.getEndTime())).count()).isEqualTo(1);
+    assertThat(personWinner.getPreviousPositions().stream()
+        .filter(f -> Objects.isNull(f.getEndTime()) && Objects.isNull(f.getPositionUuid())).count())
+            .isEqualTo(1);
+
+    personLoser = assignPositionPerson(positionLoser, personLoser);
```

Comment:
It's probably neater to use `final Person personLoser2 = …` etc.

Reply:
Used final for variables.